src/brogue/Time.c

```diff
@@ -2428,7 +2411,8 @@ void playerTurnEnded() {
                 refreshWaypoint(rogue.wpRefreshTicker);
             }
 
-            for (monst = monsters->nextCreature; (monst != NULL) && (rogue.gameHasEnded == false); monst = monst->nextCreature) {
+            for (creatureIterator it = iterateCreatures(&monsters); hasNextCreature(it);) {
```

Comment:
Missing `rogue.gameHasEnded == false` test - is it necessary?

Reply:
Fixed; I probably just mindlessly rewrote this without noticing that there was an extra condition. It's been restored.